src/components/StoryView/Story.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import {
+  MultiStoryViewContainer,
+  SingleStoryView,
+  StoryContainerProps,
+  StoryRef,
+} from '.';
+
+const Story = React.forwardRef<StoryRef, StoryContainerProps>(
+  ({ ...props }, ref) => {
+    if (props.extended) {
+      return <SingleStoryView {...props} ref={ref} />;
```

Comment:
@sagar-simformsolutions Extended prop is for single story only not for multi-story, so this should be revised as follows

`props?.extended ? <SingleStoryView {...props} ref={ref} /> : <StoryContainer {...props} ref={ref} />`